apps/mobile/src/components/widgets/widget.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I'm not sure if I should move `widgets` to the UI library? 

I can as I work on building them out but we don't really share them on web yet. I could to enforce a cleaner pattern of UI in the UI lib

Reply:
What is a `widget` exactly? It looks like just a `Box` component? Anything special about it?

Reply:
Nope, nothing special just a `Box` but I thought if we are building a few of them better to have a wrapper than repeat the box and padding.

I can also get rid of it?

Reply:
I guess it's similar to `Cell` in that it helps match design to dev. 

`Box`s are essentially `div`s anyway 😄

Reply:
I did the same with `Avatar` bc the code was being repeated, hoping we can expand on what I relocated to the UI lib too.

Reply:
Sounds good. We can move fast but take notes as we go and implement more re-usable patterns. Even if we have simple components not doing much I think it's nice to read code like this:
```
 <Widget>
   {something.map(thing => 
       <Cell name={thing.name}/>
 }
</Widget>

```

Reply:
I suppose with this approach, the refactor I did to keep `Cell` would make sense even though it just wrapped `TouchableOpacity` to create consistent formatting?